baseplate/__init__.py

```diff
@@ -18,13 +18,13 @@ def metrics_client_from_config(raw_config):
     This expects two configuration options:
 
     ``metrics.namespace``
-        The root key to namespace all metrics in this application under.
+        The root key to prefix all metrics in this application with.
     ``metrics.endpoint``
         A ``host:port`` pair, e.g. ``localhost:2014``. If an empty string, a
         client that discards all metrics will be returned.
 
-    :param dict raw_config: The app configuration which should have settings
-        for the metrics client.
+    :param dict raw_config: The application configuration which should have
```

Comment:
The main function of a baseplate application gets the application configuration passed in as a dictionary. That's what this is supposed to be. It's basically just "all of the raw config values from the INI file." Does that make sense? or should I document it somewhere?